R/expect-comparison.R

```diff
@@ -31,21 +31,27 @@ expect_compare_ <- function(
   msg <- c(
     "<" = "not strictly less than",
     "<=" = "not less than",
-    ">" = "not strictly more than",
-    ">=" = "not more than"
+    ">" = "not strictly greater than",
+    ">=" = "not greater than"
   )[[operator]]
 
+  negated_op <- switch(operator, "<" = ">", "<=" = ">=", ">" = "<", ">=" = "<=")
```

Comment:
Aren't these wrong? e.g. I think we'll get

```
expect_snapshot_failure(expect_gt(x, 1.0 * x))
```

```
    `x` is not strictly greater than 1.0 * x.
    0.0000100 - 0.0000110 = 0 < 0
```

(even if I'm reading it wrong we'll still want a test of the edge case)